packages/core/src/modules/connections/TrustPingEvents.ts

```diff
@@ -0,0 +1,22 @@
+import type { BaseEvent } from '../../agent/Events'
+import type { DidExchangeState } from './models'
+import type { ConnectionRecord } from './repository/ConnectionRecord'
+
+export enum TrustPingEventTypes {
+  TrustPingRequestEvent = 'TrustPingRequestEvent',
+  TrustPingResponseEvent = 'TrustPingResponseEvent',
```

Comment:
Received is more consistent with existing event names in AFJ:

```suggestion
  TrustPingReceivedEvent = 'TrustPingReceivedEvent',
  TrustPingResponseReceivedEvent = 'TrustPingResponseReceivedEvent',
```

Reply:
Done